lib/wrappers/genesisProtocol.ts

```diff
@@ -852,43 +854,65 @@ export class GenesisProtocolWrapper extends ContractWrapperBase implements Schem
 
     params = Object.assign({},
       {
-        boostedVotePeriodLimit: 60,
+        boostedVotePeriodLimit: 604800, // 1 week
```

Comment:
These are default values which can be override be arc.js user ?

Reply:
Yes